### DISCUSSION TITLE
欢迎大家进入 *计算机视觉* 的世界！畅所欲言

### DISCUSSION THREAD

**Discussion by Charmve** [Charmve/computer-vision-in-action#1] [Show and tell]
目前该电子书仓库还在继续更新中，会逐渐完善章节内容和与之匹配的实战代码。大家有什么问题都可以在这里 == [**Discussion**] 提出，我会尽量及时回复大家。

目前已更新的篇目我会用超链接的方式在根目录下的 README.md 目录给出，点击可直接访问。

例如，

  - 第 17 章 [跨界模型 Transformer](https://github.com/Charmve/computer-vision-in-action/blob/main/docs/3_进阶篇/chapter17-跨界模型%20Transformer/chapter17_Transformer.md)
    - 17.1 [思想和框图](https://github.com/Charmve/computer-vision-in-action/blob/main/docs/3_进阶篇/chapter17-跨界模型%20Transformer/chapter17_Transformer.md#171-思想和框图)
    - 17.2 [实现细节](https://github.com/Charmve/computer-vision-in-action/blob/main/docs/3_进阶篇/chapter17-跨界模型%20Transformer/chapter17_Transformer.md#172-实现细节)
      - [2.1 Encoder](https://github.com/Charmve/computer-vision-in-action/blob/main/docs/3_进阶篇/chapter17-跨界模型%20Transformer/chapter17_Transformer.md#1721-Encoder)
      - [2.2 Decoder](https://github.com/Charmve/computer-vision-in-action/blob/main/docs/3_进阶篇/chapter17-跨界模型%20Transformer/chapter17_Transformer.md#1722-decoder)
      - [2.3 Self-Attention](https://github.com/Charmve/computer-vision-in-action/blob/main/docs/3_进阶篇/chapter17-跨界模型%20Transformer/chapter17_Transformer.md#1723-Self-Attention)
      - [2.4 Multi-Headed Attention](https://github.com/Charmve/computer-vision-in-action/blob/main/docs/3_进阶篇/chapter17-跨界模型%20Transformer/chapter17_Transformer.md#1724-Multi-Headed-Attention)
      - [2.5 Positional Encoding](https://github.com/Charmve/computer-vision-in-action/blob/main/docs/3_进阶篇/chapter17-跨界模型%20Transformer/chapter17_Transformer.md#1725-Positional-Encoding)
    - 17.3 [应用任务和结果](https://github.com/Charmve/computer-vision-in-action/blob/main/docs/3_进阶篇/chapter17-跨界模型%20Transformer/chapter17_Transformer.md#173-应用任务和结果)
      - [3.1 NLP领域](https://github.com/Charmve/computer-vision-in-action/blob/main/docs/3_进阶篇/chapter17-跨界模型%20Transformer/chapter17_Transformer.md#1731-NLP领域)
      - [3.2 CV领域](https://github.com/Charmve/computer-vision-in-action/blob/main/docs/3_进阶篇/chapter17-跨界模型%20Transformer/chapter17_Transformer.md#1732-CV领域)
        - [3.2.1 检测DETR](https://github.com/Charmve/computer-vision-in-action/blob/main/docs/3_进阶篇/chapter17-跨界模型%20Transformer/chapter17_Transformer.md#17321-检测DETR)
        - [3.2.2 分类ViT](https://github.com/Charmve/computer-vision-in-action/blob/main/docs/3_进阶篇/chapter17-跨界模型%20Transformer/chapter17_Transformer.md#17322-分类ViT)
        - [3.2.3 分割SETR](https://github.com/Charmve/computer-vision-in-action/blob/main/docs/3_进阶篇/chapter17-跨界模型%20Transformer/chapter17_Transformer.md#17323-分割setr)
        - [3.2.4 Deformable-DETR](https://github.com/Charmve/computer-vision-in-action/blob/main/docs/3_进阶篇/chapter17-跨界模型%20Transformer/chapter17_Transformer.md#17324-Deformable-DETR)
    - 17.4 [优点及分析](https://github.com/Charmve/computer-vision-in-action/blob/main/docs/3_进阶篇/chapter17-跨界模型%20Transformer/chapter17_Transformer.md#174-优点及分析)
    - 17.5 [缺点及分析](https://github.com/Charmve/computer-vision-in-action/blob/main/docs/3_进阶篇/chapter17-跨界模型%20Transformer/chapter17_Transformer.md#175-缺点及分析)
    - [参考文献](https://github.com/Charmve/computer-vision-in-action/blob/main/docs/3_进阶篇/chapter17-跨界模型%20Transformer/chapter17_Transformer.md#176-参考文献)

由于项目还没完善，鼓励大家优先 Star，如果大家想参与进来，也欢迎大家提 Pull Request。

目前也可以在线阅读电子书了，源码和notebook在主页对应文件夹下。

👉 https://charmve.github.io/computer-vision-in-action

**Comment by Charmve** [comment]
- 完整的学习文档可以在 [主页](https://github.com/Charmve/computer-vision-in-action) 看到;
- 完整的L0CV系统架构可以在 [L0CV](https://github.com/Charmve/computer-vision-in-action/tree/main/L0CV)看到;
- 更多精彩好玩的活动可在 [L0CV-Universe](https://github.com/Charmve/computer-vision-in-action/tree/main/L0CV-Universe) 找到, 目前主要有 Challenges、Meetup、DemoDay 三个；

**Comment by fanermeier** [comment]
您好！您发给我的信件已经收到。联系电话：18290290169
